pages/_lang/generate.vue

```diff
@@ -451,6 +471,54 @@ export default class extends Vue {
     this.manifest$ = { ...this.manifest };
   }
 
+  public onClickDownloadAll() {
+
+    // local azure function
+    this.zipRequested = true;
+    const images: generator.Icon[] = [];
+    const length = this.icons.length;
+    // prevent the passing of the observer objects in the body.
+    for (let i = 0; i < length; i++) {
+      images.push({ ...this.icons[i] })
+    }
+
+    axios.post("https://azure-express-zip-creator.azurewebsites.net/api", JSON.stringify({ images }), {
+      "method": "POST",
+      "responseType": "blob",
+      "headers": {
+        "content-type": "application/json"
+      }
+    })
+    .then(async res => {
+      if (window.chooseFileSystemEntries) {
+        const fsOpts = {
+          type: "save-file",
+          accepts: [
+            {
+              description: "PWA Builder Image Zip",
+              extensions: ["zip"],
+              mimeTypes: ["application/zip"]
+            }
+          ]
+        };
+        const fileHandle = await window.chooseFileSystemEntries(fsOpts);
+        // Create a FileSystemWritableFileStream to write to.
+        const writable = await fileHandle.createWritable();
+        // Write the contents of the file to the stream.
+        await writable.write(res);
+        // Close the file and write the contents to disk.
+        await writable.close();
```

Comment:
not to be supported until chrome 83. can verify with chrome-canary

Reply:
Cool to support that!